akka-docs/rst/java/code/docs/stream/GraphDSLDocTest.java

```diff
@@ -50,25 +50,24 @@ public void demonstrateBuildSimpleGraph() throws Exception {
     //#simple-graph-dsl
     final Source<Integer, NotUsed> in = Source.from(Arrays.asList(1, 2, 3, 4, 5));
     final Sink<List<String>, CompletionStage<List<String>>> sink = Sink.head();
-    final Sink<List<Integer>, CompletionStage<List<Integer>>> sink2 = Sink.head();
     final Flow<Integer, Integer, NotUsed> f1 = Flow.of(Integer.class).map(elem -> elem + 10);
     final Flow<Integer, Integer, NotUsed> f2 = Flow.of(Integer.class).map(elem -> elem + 20);
     final Flow<Integer, String, NotUsed> f3 = Flow.of(Integer.class).map(elem -> elem.toString());
     final Flow<Integer, Integer, NotUsed> f4 = Flow.of(Integer.class).map(elem -> elem + 30);
 
     final RunnableGraph<CompletionStage<List<String>>> result =
-      RunnableGraph.<CompletionStage<List<String>>>fromGraph(
+      RunnableGraph.fromGraph(
         GraphDSL
           .create(
-            sink,
-            (builder, out) -> {
+            sink,                // previously created sink
+            (builder, out) -> {  // variables: builder (GraphDSL.Builder) and out (SinkShape)
```

Comment:
hm, maybe rather "out is the `sink`s out port (we need to use its shape below)"?